_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
-
-* [Our interactive introduction to Scala on scala-exercises.com](https://www.scala-exercises.org/scala_tutorial/terms_and_types), or
-* [Scastie](https://scastie.scala-lang.org/), Scala in the browser, with access to all Scala compilers and all published libraries!
-
-## Setting up and getting started with Scala
-
-### If you prefer working in an IDE...
-
-IntelliJ is the most commonly-used IDE by Scala developers. In this tutorial,
-we'll walk you through downloading and setting up IntelliJ with the Scala
-plugin, and we'll get you started with your first Scala project, complete with
-unit tests!
-
-* [Getting Started with Scala in IntelliJ](/getting-started/intellij-track/getting-started-with-scala-in-intellij.html)
-* [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html)
-* [Testing Scala in IntelliJ with ScalaTest](/getting-started/intellij-track/testing-scala-in-intellij-with-scalatest.html)
-
-
-### If you prefer working on the command line...
-
-If you prefer using a text editor like emacs, Vim, Atom, or Sublime Text, then
-the best way to compile, test, and run Scala code is by using _sbt_, Scala's build
-tool.
-
-* [Getting Started with Scala and sbt on the Command Line](/getting-started/sbt-track/getting-started-with-scala-and-sbt-on-the-command-line.html)
-* [Testing Scala with sbt and ScalaTest on the Command Line](/getting-started/sbt-track/testing-scala-with-sbt-on-the-command-line.html)
-
-<!-- sbt is the easiest way to ensure that your Scala project is reproducible;
-you specify a Scala version, any libraries you depend on, and sbt takes care of
-the rest, so it's as easy as possible for someone else to compile and run your
-Scala project. -->
 
 ## Next Steps
 Once you've finished these tutorials, check out:
 
-* [The Tour of Scala](/tour/tour-of-scala.html) for bite-sized introductions to Scala's features.
 * [The Scala Book](/overviews/scala-book/introduction.html), which provides a set of short lessons introducing Scala’s main features.
+* [The Tour of Scala](/tour/tour-of-scala.html) for bite-sized introductions to Scala's features.
 * [Learning Resources](/learn.html), which includes online interactive tutorials and courses.
 * [Our list of some popular Scala books](/books.html).
 
 ## Getting Help
 There are a multitude of mailing lists and real-time chat channels in case you want to quickly connect with other Scala users. Check out our [community](https://scala-lang.org/community/) page for a list of these resources, and for where to reach out for help.
+
+<!-- Hidden elements whose content are used to provide OS-specific download instructions.
+ -- This is handled in `resources/js/functions.js`.
+ -->
+    
+<div style="display:none" id="stepOne-linux">
+       <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs setup </code> <br>
```

Comment:
Where should this command be invoked from? What do we do with the file `cs` after the `setup` command is completed? Should we keep it?

Reply:
I don't know what to say about that... you can run it from everywhere, and you can delete it at the end, or keep it..